### PR TITLE
Replace deprecated node-sass with dart-sass

### PR DESCRIPTION
**Pull request by Morred**
### Description of Change

Replace node-sass with dart-sass. This should be merged before https://github.com/reaktor/library/pull/7

### Related Issue

node-sass is deprecated

### Checklist

- [x] Ran `npm run lint` and updated code style accordingly
- [x] `npm run test` passes -> tests are already failing on the main branch so won't be fixed here
- [x] PR has a description and all contributors/stakeholder are noted/cc'ed
- [ ] ~tests are updated and/or added to cover new code~
- [ ] ~relevant documentation is changed and/or added~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,8 +52,8 @@
   },
   "scripts": {
     "start": "node server/index.js",
-    "build:main": "node-sass --include-path custom/styles --include-path styles/partials --source-map public/css/style.css.map --source-map-contents styles/style.scss public/css/style.css",
-    "build:inline": "node-sass --include-path custom/styles --include-path styles/partials styles/errors.scss public/css/errors.css",
+    "build:main": "sass --load-path custom/styles --load-path styles/partials --source-map styles/style.scss public/css/style.css",
```
Comment (EsaNuurtamo) [inline package.json]: It seems that the source maps are generated automatically by dart sass: https://sass-lang.com/documentation/cli/dart-sass

The --source-map does not seem to exist anymore

**Review by EsaNuurtamo — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (Morred) [reply package.json]: Updated, thanks.

**Review by tommi-h — APPROVED** [review]
Looks good! Thank you.

**Pull request merged**